testplan/header.go

```diff
@@ -25,24 +25,11 @@ ifndef::env-github[]
 endif::[]
 :sectanchors:
 :doctype: book
-:revnumber: 0.1
-:revdate: %s
```

Comment:
This is just using fmt.Sprintf instead of a proper template, so if you remove these verbs, you should also remove them from the Sprintf call on line 67. Or replace the whole thing with a proper template; I was just being lazy with the Sprintf.